release_build_files/readme.md

```diff
@@ -337,40 +337,41 @@ information).
 
 #### OS X libraries
 
-For OS X (Darwin), library versions are only provided for the 64-bit (x86_64)
-platform. See the table above (in the "Linux libraries" section) for the list of
-library dependencies. XCFrameworks are also provided for convenience.
+For OS X (Darwin), library versions are provided for both Intel (x86_64) and ARM
+(arm64) platforms, as well as universal libraries. See the table above (in the
+"Linux libraries" section) for the list of library dependencies. Frameworks are
+also provided for your convenience.
 
-Feature                         | Required XCFrameworks
+Feature                         | Required Frameworks
 ------------------------------- | ----------------------------------
-Firebase Authentication         | firebase_auth.xcframework
-|                               | firebase.xcframework
-Cloud Firestore                 | firebase_firestore.xcframework
-|                               | firebase_auth.xcframework
-|                               | firebase.xcframework
-Firebase Functions              | firebase_functions.xcframework
-|                               | firebase_auth.xcframework (optional)
-|                               | firebase.xcframework
-Firebase Realtime Database      | firebase_database.xcframework
-|                               | firebase_auth.xcframework
-|                               | firebase.xcframework
-Firebase Remote Config          | firebase_remote_config.xcframework
-|                               | firebase.xcframework
-Firebase Storage                | firebase_storage.xcframework
-|                               | firebase_auth.xcframework
-|                               | firebase.xcframework
-Firebase AdMob (stub)           | firebase_admob.xcframework
-|                               | firebase.xcframework
-Firebase Analytics (stub)       | firebase_analytics.xcframework
-|                               | firebase.xcframework
-Firebase Dynamic Links (stub)   | firebase_dynamic_links.xcframework
-|                               | firebase.xcframework
-Firebase Installations (stub)   | firebase_installations.xcframework
-|                               | firebase.xcframework
-Firebase Cloud Messaging (stub) | firebase_messaging.xcframework
-|                               | firebase.xcframework
-
-The provided libraries have been tested using Xcode 12.0. When building C++
+Firebase Authentication         | firebase_auth.framework
```

Comment:
We're not providing xcframeworks anymore?  I thought they were the new thing, not the old thing..?

Reply:
We are for iOS. We aren't for OSX, and never were. This was in error.

Reply:
Ahhh, I see.